Makefile

```diff
@@ -1,15 +1,38 @@
 .PHONY: all check-style unittest generate build clean build-image operator-sdk yaml
 
+# Current Operator version - used for bundle
+VERSION ?= 1.6.0
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# Image URL to use all building/pushing image targets
 OPERATOR_IMAGE ?= mattermost/mattermost-operator:test
-SDK_VERSION = v0.19.0
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+SDK_VERSION = v1.0.1
 MACHINE = $(shell uname -m)
 BUILD_IMAGE = golang:1.14.6
-BASE_IMAGE = alpine:3.12
+BASE_IMAGE = gcr.io/distroless/static:nonroot
```

Comment:
Interesting. I am just going to not this for @cpanato in case he didn't happen to see the image change. I don't know much about this image, but I imagine it is more secure than the alpine image we were using.